### ISSUE TITLE
Add "effected rows" count to update query

### ISSUE THREAD

**Issue by shaloba** [tortoise/tortoise-orm#126]
Hi,

Thanks for lovely ORM. 
it would be nice if the update method return the number of the effected rows after update query

for example: 

```
result = return await Users.filter(user_id=user_id).update(is_active=False)
# result value is None
```

to verify the update query succeeded i need to query the user again and retrieve the data

Thanks in advanced!

**Comment by grigi** [comment]
This is a very reasonable suggestion, the `update` equivalent of #68

**Comment by AEnterprise** [comment]
is there any update to this? would be really nice to have

**Comment by grigi** [comment]
No update unfortunately. I forgot about this.
I will have to investigate how to get update/delete counters from the various DB drivers.

**Comment by AEnterprise** [comment]
iirc most (if not all) return this count from the .execute() methods

**Comment by grigi** [comment]
Nope, the drivers all do something different. Remember DB-API covers sync, not async.

But it's done :-)

**Comment by grigi** [comment]
Hi, I released v0.15.7, please test if this issue is resolved :-)

**Comment by AEnterprise** [comment]
works perfect, thanks a ton!
